test/pip-rehash.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+@test "pip-rehash set PYENV_REHASH_COMMAND right when using 'pip'" {
+  PYENV_COMMAND="pip"
+  source "${BATS_TEST_DIRNAME}/../pyenv.d/exec/pip-rehash.bash"
```

Comment:
The tests should probably rather check if `pyenv-rehash` is called after invoking a stub executable with the corresponding name.

This `source` command does not run the hook in the same environment as it would normally run in so it can behave differently than in real use and we won't see that.

Reply:
Done